dictionaries/en_US/checksum.txt

```diff
@@ -1,4 +1,4 @@
-3378f52c51216887fc3564b878ac8372f7a4d380  ./en_US.trie.gz
```

Comment:
Please remove this file from the PR.
and the change to `en_US.trie.gz`.

Reply:
Ah sorry, yeah that accidentally got included, it should be removed now.